src/blog/2023/08/aws-marketplace-announce.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FlowFuse is now available on AWS Marketplace
+subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
+authors: ["ian-skerrett"]
+image: blog/2023/08/images/UNS/UNS-Headline.svg
```

Comment:
Consider an image from unsplash if you want to publish today: https://unsplash.com/photos/M2524ncJQ40